transcriptic/cli.py

```diff
@@ -444,6 +444,12 @@ def compile_cmd(protocol_name, args):
     help='Project id or name context for configuring the protocol. Use '
          '`transcriptic projects` command to list existing projects.'
 )
+@click.option(
+    '--title', '-t',
+    metavar='RUN_TITLE',
+    required=False,
+    help='If specified, will apply custom title to run created.'
```

Comment:
hmm we should probably specify what's the created format somewhere. Perhaps here, or mention its a prefix?